2025-10-21 21:12:08,805 : INFO : root : Loading config.
2025-10-21 21:12:08,810 : INFO : YamlMultifileConfigLoader : Machine config file #1: config.yaml
2025-10-21 21:12:08,810 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\fdca14d0c9d658ab8200d57120800af5.mpf_cache
2025-10-21 21:12:08,916 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\5dec96b1547c4adf5bdd3d1715e74bc0.mpf_cache
2025-10-21 21:12:08,916 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\202a5840610e95051f9f473dfa660cc0.mpf_cache
2025-10-21 21:12:08,917 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\5dec96b1547c4adf5bdd3d1715e74bc0.mpf_cache
2025-10-21 21:12:08,918 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\2cd13a2d3f375e00ab01d6d09a85d8e7.mpf_cache
2025-10-21 21:12:08,919 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\9220b44b97b91972524500b81035d790.mpf_cache
2025-10-21 21:12:08,919 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\dab29d17bbe6500d014c71b0a57bd9b0.mpf_cache
2025-10-21 21:12:08,920 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\b17152efa55a24f0424602723f8d3921.mpf_cache
2025-10-21 21:12:08,920 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\e87fc0e7d2f8a851b5fa3f604e0d0c6e.mpf_cache
2025-10-21 21:12:08,920 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\196ac76b6cab3a9880804c826cfbcb5c.mpf_cache
2025-10-21 21:12:08,922 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\834292dd7853e12a66732dc89828c828.mpf_cache
2025-10-21 21:12:08,922 : WARNING : ConfigProcessor : Config file in cache changed: C:\Users\antho\nightpin\shows\.\graveyard_lights.yaml
2025-10-21 21:12:08,923 : INFO : ConfigProcessor : Loading config from file C:\Users\antho\nightpin\shows\.\graveyard_lights.yaml.
2025-10-21 21:12:08,938 : INFO : ConfigProcessor : Loading config: C:\Users\antho\nightpin\shows\.\graveyard_lights.yaml
2025-10-21 21:12:08,939 : INFO : ConfigProcessor : Config file cache created: C:\Users\antho\AppData\Local\Temp\ae2be9c5579852c305357e207df97e8b.mpf_cache
2025-10-21 21:12:08,939 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\46f8fa3e41dfeeb8232af2f03a13b6c4.mpf_cache
2025-10-21 21:12:08,940 : WARNING : ConfigProcessor : Config file in cache changed: C:\Users\antho\nightpin\shows\.\house_lights.yaml
2025-10-21 21:12:08,940 : INFO : ConfigProcessor : Loading config from file C:\Users\antho\nightpin\shows\.\house_lights.yaml.
2025-10-21 21:12:08,952 : INFO : ConfigProcessor : Loading config: C:\Users\antho\nightpin\shows\.\house_lights.yaml
2025-10-21 21:12:08,954 : INFO : ConfigProcessor : Config file cache created: C:\Users\antho\AppData\Local\Temp\c67e9c175da6c060302945929b27b878.mpf_cache
2025-10-21 21:12:08,954 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\1c965aadbb61a7e709e8884fad17ac0b.mpf_cache
2025-10-21 21:12:08,955 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\60ca9d662df3e70e96f6d7ec0f4491aa.mpf_cache
2025-10-21 21:12:08,963 : INFO : Machine : Mission Pinball Framework Core Engine v0.80.0.dev11
2025-10-21 21:12:08,964 : INFO : Machine : Command line arguments: {'no_load_cache': False, 'create_config_cache': True, 'bcp': False, 'configfile': ['config.yaml'], 'force_assets_load': False, 'platform_integration_test': False, 'jsonlogging': False, 'logfile': 'logs\\2025-10-21-21-12-08-mpf-LAPTOP-MPGP6R69.log', 'pause': False, 'production': False, 'text_ui': False, 'loglevel': 15, 'consoleloglevel': 20, 'force_platform': 'virtual', 'syslog_address': None, 'full_logfile_path': 'C:\\Users\\antho\\nightpin\\logs\\2025-10-21-21-12-08-mpf-LAPTOP-MPGP6R69.log'}
2025-10-21 21:12:08,964 : INFO : Machine : MPF path: C:\Users\antho\mpf\Lib\site-packages\mpf
2025-10-21 21:12:08,964 : INFO : Machine : Machine path: C:\Users\antho\nightpin
2025-10-21 21:12:08,964 : INFO : Machine : Platform: win32
2025-10-21 21:12:08,964 : INFO : Machine : Python executable location: C:\Users\antho\mpf\Scripts\python.exe
2025-10-21 21:12:08,964 : INFO : Machine : Python version: 3.12.7 (64-bit)
2025-10-21 21:12:08,966 : INFO : Machine : Initializing MPF...
2025-10-21 21:12:09,061 : INFO : EventManager : Event: ======'machine_var_credits_string'====== Args={'value': 'FREE PLAY', 'prev_value': None, 'change': True}
2025-10-21 21:12:09,090 : INFO : EventManager : Event: ======'machine_var_master_volume'====== Args={'value': 0.5, 'prev_value': None, 'change': True}
2025-10-21 21:12:09,090 : INFO : EventManager : Event: ======'machine_var_mpf_version'====== Args={'value': 'Mission Pinball Framework v0.80.0.dev11', 'prev_value': None, 'change': True}
2025-10-21 21:12:09,090 : INFO : EventManager : Event: ======'machine_var_mpf_extended_version'====== Args={'value': 'Mission Pinball Framework v0.80.0.dev11, Config version:6, Show version: 6, BCP version:1.1', 'prev_value': None, 'change': True}
2025-10-21 21:12:09,090 : INFO : EventManager : Event: ======'machine_var_python_version'====== Args={'value': '3.12.7', 'prev_value': None, 'change': True}
2025-10-21 21:12:09,249 : INFO : EventManager : Event: ======'machine_var_platform'====== Args={'value': 'Windows-11-10.0.26100-SP0', 'prev_value': None, 'change': True}
2025-10-21 21:12:09,249 : INFO : EventManager : Event: ======'machine_var_platform_system'====== Args={'value': 'Windows', 'prev_value': None, 'change': True}
2025-10-21 21:12:09,249 : INFO : EventManager : Event: ======'machine_var_platform_release'====== Args={'value': '11', 'prev_value': None, 'change': True}
2025-10-21 21:12:09,249 : INFO : EventManager : Event: ======'machine_var_platform_version'====== Args={'value': '10.0.26100', 'prev_value': None, 'change': True}
2025-10-21 21:12:09,249 : INFO : EventManager : Event: ======'machine_var_platform_machine'====== Args={'value': 'AMD64', 'prev_value': None, 'change': True}
2025-10-21 21:12:09,249 : INFO : EventManager : Event: ======'machine_var_log_file_path'====== Args={'value': 'C:\\Users\\antho\\nightpin\\logs\\2025-10-21-21-12-08-mpf-LAPTOP-MPGP6R69.log', 'prev_value': None, 'change': True}
2025-10-21 21:12:09,249 : INFO : EventManager : Event: ======'init_phase_1'====== Args={}
2025-10-21 21:12:09,281 : INFO : EventManager : Event: ======'drop_target_bank_right_bank_up'====== Args={}
2025-10-21 21:12:09,282 : INFO : EventManager : Event: ======'drop_target_bank_left_bank_up'====== Args={}
2025-10-21 21:12:09,282 : INFO : EventManager : Event: ======'init_phase_2'====== Args={}
2025-10-21 21:12:09,289 : INFO : ball_device.bd_trough : BCH: Initial count: 0
2025-10-21 21:12:09,289 : INFO : ball_device.bd_trough : State: idle
2025-10-21 21:12:09,289 : INFO : ball_device.bd_saucer : BCH: Initial count: 0
2025-10-21 21:12:09,289 : INFO : ball_device.bd_saucer : State: idle
2025-10-21 21:12:09,298 : INFO : EventManager : Event: ======'init_phase_3'====== Args={}
2025-10-21 21:12:09,306 : ERROR : asyncio : Exception in callback EventManager._queue_task_done(<Task finishe... 0 duration')>)
handle: <Handle EventManager._queue_task_done(<Task finishe... 0 duration')>)>
Traceback (most recent call last):
  File "C:\Users\antho\AppData\Local\Programs\Python\Python312\Lib\asyncio\events.py", line 88, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\events.py", line 795, in _queue_task_done
    future.result()
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\events.py", line 708, in _run_handlers_sequential
    handler.callback(queue=queue, **merged_kwargs)
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\show_controller.py", line 53, in _load_shows
    self.machine.shows[show_name].load(show_config)
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\assets\show.py", line 160, in load
    self._show_validation_error("Step {} has 0 duration".format(step_num), 7)
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\assets\show.py", line 184, in _show_validation_error
    raise ConfigFileError('"{}" >> {}'.format(self.name, msg), error_code, "show", self.name)
mpf.exceptions.config_file_error.ConfigFileError: Config File Error in show: "graveyard_reach_house" >> Step 1 has 0 duration Context: graveyard_reach_house Error Code: CFE-show-7 (https://missionpinball.org/logs)
2025-10-21 21:12:09,309 : INFO : Machine : Starting the main run loop with active modes: []
2025-10-21 21:12:09,309 : WARNING : Machine : Attract mode is not active, game will not be playable. Please check your attract mode configuration.
2025-10-21 21:12:09,309 : INFO : Machine : Shutting down...
2025-10-21 21:12:09,309 : INFO : EventManager : Event: ======'shutdown'====== Args={}
2025-10-21 21:12:10,318 : ERROR : Machine : Runtime Exception
Traceback (most recent call last):
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\machine.py", line 810, in _run_loop
    raise self._exception['exception']
  File "C:\Users\antho\AppData\Local\Programs\Python\Python312\Lib\asyncio\events.py", line 88, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\events.py", line 795, in _queue_task_done
    future.result()
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\events.py", line 708, in _run_handlers_sequential
    handler.callback(queue=queue, **merged_kwargs)
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\show_controller.py", line 53, in _load_shows
    self.machine.shows[show_name].load(show_config)
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\assets\show.py", line 160, in load
    self._show_validation_error("Step {} has 0 duration".format(step_num), 7)
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\assets\show.py", line 184, in _show_validation_error
    raise ConfigFileError('"{}" >> {}'.format(self.name, msg), error_code, "show", self.name)
mpf.exceptions.config_file_error.ConfigFileError: Config File Error in show: "graveyard_reach_house" >> Step 1 has 0 duration Context: graveyard_reach_house Error Code: CFE-show-7 (https://missionpinball.org/logs)
2025-10-21 21:12:10,320 : INFO : root : MPF run loop ended.
